icu4c/source/common/brkeng.h

```diff
@@ -77,6 +77,11 @@ class LanguageBreakEngine : public UMemory {
                               UVector32 &foundBreaks,
                               UErrorCode &status) const = 0;
 
+  void enablePhraseBreaking(UBool enable);
```

Comment:
It is ok to change the interface of LanguageBreakEngine since it is not considered as API. However, the design of LanguageBreakEngine is it need to be immutable and the internal state not changed after the construction because they will be shared by multiple thread. Adding a mutable method like enablePhraseBreaking() to change internal state is not acceptable.

Reply:
Remove this function